bindgen/src/gen_cs/mod.rs

```diff
@@ -61,6 +63,14 @@ impl Config {
             "uniffi".into()
         }
     }
+
+    pub fn namespace(&self) -> String {
```

Comment:
Initially I tried that and the compiler yell that the return type must implement Display (Probably because the Template derivation).
Anyway you are right and I came to a conclusion that we don't need this function at all. Removed it and used directly the field in the template.